src/backend/aspen/api/schemas/samples.py

```diff
@@ -84,7 +105,6 @@ class Config:
     collection_location: LocationResponse
     czb_failed_genome_recovery: bool
     gisaid: Optional[SampleGisaidResponse]
-    lineage: Optional[SampleLineageResponse]
```

Comment:
Keep in mind that removing this field from the backend in the same deployment that updates the frontend to stop using it could make the UI behave strangely while we update our service to the latest version. I don't think we need to stress it for every change, but I want to make sure we're making conscious decisions to accept these tradeoffs.

Reply:
yeah in the interest of time i was trying to do all this at once, i personally think it would be an ok tradeoff this one time? I was thinking that the update would be fairly fast, so the amount of time there would potentially be funkiness would be an ok tradeoff? I do agree that we should minimize this in the future.

Reply:
yeah, I think it's ok.

Reply:
in this case i think it's ok, i was doing this all together because of the time constraints to get it finished before the sprint ended. I think i would accept UI funkiness while the backend is updating for this, but i agree with trying to minimize this in the future, and i will take more time to do that next time.

Reply:
my original plan was to do this in steps but the time got away from me a bit and i wanted to get this in on time. I will definitely take more care in the future around this.